scalableminds/webknossos
# add admins and dataset managers to dataset access list

### Steps to test:
- click the plus sign left of dataset name in dataset list
- should list admins and team managers of the dataset’s organization (if the requesting user is allowed to see them)

### Issues:
- fixes #4855

------
